maskrcnn_benchmark/modeling/roi_heads/mask_head/roi_mask_feature_extractors.py

```diff
@@ -32,17 +34,17 @@ def __init__(self, cfg):
         input_size = cfg.MODEL.BACKBONE.OUT_CHANNELS
         self.pooler = pooler
 
+        use_gn = cfg.MODEL.ROI_MASK_HEAD.USE_GN
         layers = cfg.MODEL.ROI_MASK_HEAD.CONV_LAYERS
+        dilation = cfg.MODEL.ROI_MASK_HEAD.DILATION
 
         next_feature = input_size
         self.blocks = []
         for layer_idx, layer_features in enumerate(layers, 1):
             layer_name = "mask_fcn{}".format(layer_idx)
-            module = Conv2d(next_feature, layer_features, 3, stride=1, padding=1)
-            # Caffe2 implementation uses MSRAFill, which in fact
-            # corresponds to kaiming_normal_ in PyTorch
-            nn.init.kaiming_normal_(module.weight, mode="fan_out", nonlinearity="relu")
-            nn.init.constant_(module.bias, 0)
+            module = make_conv3x3(next_feature, layer_features, 
```

Comment:
This will currently break loading of previously saved models, because you always return a `nn.Sequential` here.
Could you instead return a `nn.Conv2d` if it's not `gn`?
Also, as mentioned, there is a reduntant ReLU in `make_conv3x3`.

Reply:
understand. i will keep it as before